src/main/java/com/rakovets/course/javabasics/practice/loops/Task01.java

```diff
@@ -31,6 +31,11 @@ public static void main(String[] args) {
     static String generateNumbersColumn(int numberRows) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+
+        StringBuilder numbersColumn = new StringBuilder();
```

Comment:
Why are you using `StringBuilder`?)

Reply:
To answer your question - the String can't be changed once it is created, but StringBuilder can be expanded. It does not create a new object in the memory and that's why doesn’t have the extra garbage left over. So StringBuilder is faster.

As to my knowlegde, the StringBuilder class is recommended for use with an unknown number of operations and changes on lines during program execution and also when the application is expected to do many such operations. Therefore, it seems to me that StringBuilder is more suitable for loops.